xml_converter/src/xml_converter.cpp

```diff
@@ -218,11 +218,11 @@ void convert_taco_directory(string directory) {
 
 int main() {
     convert_taco_directory("./packs/tw_ALL_IN_ONE");
```

Comment:
if we want to remove these files we should just change it to look for all the files in a folder.

Reply:
I think I will do this change now so it doesn't continue to be an issue when I commit